src/com/palmergames/bukkit/towny/tasks/DeleteFileTask.java

```diff
@@ -0,0 +1,21 @@
+package com.palmergames.bukkit.towny.tasks;
+
+import com.palmergames.util.FileMgmt;
+
+import java.io.File;
+
+public class DeleteFileTask implements Runnable {
```

Comment:
I feel like the `PermanentDeleteFileTask` and `DeleteFIleTask` can really be merged. Just have a boolean in the constructor to indicate whether it's a permanent delete or not and then operate based on that.